<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>VectorSaveDialog</class>
  <widget class="QDialog" name="VectorSaveDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>437</width>
        <height>345</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Save Vectors to Disk</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item rowspan="1" row="1" column="0" colspan="3" >
        <widget class="Q3ListBox" name="_vectorList" >
          <property name="selectionMode" >
            <enum>Q3ListBox::Extended</enum>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="0" column="0" colspan="3" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Select one or more vectors and click &lt;i>Save&lt;/i> to save them to a text file.</string>
          </property>
          <property name="textFormat" >
            <enum>Qt::RichText</enum>
          </property>
          <property name="scaledContents" >
            <bool>true</bool>
          </property>
          <property name="alignment" >
            <set>Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="2" >
        <widget class="QPushButton" name="pushButton2" >
          <property name="text" >
            <string>&amp;Close</string>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <spacer name="spacer1" >
          <property name="sizeHint" >
            <size>
              <width>252</width>
              <height>31</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="3" column="1" >
        <widget class="QPushButton" name="_saveButton" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <property name="text" >
            <string>&amp;Save...</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="3" >
        <widget class="KComboBox" name="_multiOptions" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <item>
            <property name="text" >
              <string>Truncate long vectors</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Interpolate short vectors</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Save in separate files</string>
            </property>
          </item>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header>kcombobox.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>_vectorList</tabstop>
    <tabstop>_multiOptions</tabstop>
    <tabstop>_saveButton</tabstop>
    <tabstop>pushButton2</tabstop>
  </tabstops>
  <includes>
    <include location="local" >kcombobox.h</include>
  </includes>
  <connections>
    <connection>
      <sender>pushButton2</sender>
      <signal>clicked()</signal>
      <receiver>VectorSaveDialog</receiver>
      <slot>hide()</slot>
    </connection>
    <connection>
      <sender>_saveButton</sender>
      <signal>clicked()</signal>
      <receiver>VectorSaveDialog</receiver>
      <slot>save()</slot>
    </connection>
    <connection>
      <sender>_vectorList</sender>
      <signal>selectionChanged()</signal>
      <receiver>VectorSaveDialog</receiver>
      <slot>selectionChanged()</slot>
    </connection>
    <connection>
      <sender>_vectorList</sender>
      <signal>doubleClicked(Q3ListBoxItem*)</signal>
      <receiver>VectorSaveDialog</receiver>
      <slot>save()</slot>
    </connection>
  </connections>
</ui>
